src/Sylius/Bundle/UiBundle/Resources/private/js/form-semantic-select.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+(function ( $ ) {
+    'use strict';
+
+    $(document).ready(function() {
+        $('.ui.fluid.search.selection.dropdown').dropdown({
+            delay: {
+                search: 500,
+            },
+            apiSettings: {
+                action: 'get taxons',
+                dataType: 'JSON',
+                data:  { 
+                    criteria: { name: {type: 'contains', value: '' } }
```

Comment:
Missing space before ``type:``.

> PRO HINT

You can copy-paste doubled space from my line 21 here XD